index.html

```diff
@@ -741,6 +738,40 @@
         </section>
 
         <section>
+          <h2 id='ldpcvget'>HTTP GET</h2>
+          <p>
+            An <a>LDPCv</a> MUST respond to <code>GET Accept: application/link-format</code> as
+            indicated in [[!RFC7089]]
+            <a href='https://tools.ietf.org/html/rfc7089#section-5'>section 5</a> and specified in
+            [[!RFC6690]] <a href='https://tools.ietf.org/html/rfc6690#section-7.3'>section 7.3</a>.
+          </p>
+          <p>
+            An implementation SHOULD respond with the <code>Allow</code> header as outlined in 
+            <a href='#ldpcvoptions'></a>.
+          </p>
+          <p>
+            An implementation MUST respond with a <code>Content-Type</code> header.
+          </p>
+          <p>
+            If an <a>LDPCv</a> supports <code>POST</code>, then it MUST respond with 
+            <code>Accept-Post</code>. If <code>POST</code> is supported but bodies are not
+            accepted, then the <a>LDPCv</a> MUST respond with <code>Accept-Post: */*; p=0.0</code>.
+          </p>
+          <p>
+            An implementation MUST respond with <code>Accept-Patch</code> if <code>PATCH</code>
```

Comment:
I am not exactly sure what we would expect with a `PATCH` on the TimeMap (LDPCv)... but if we continue to support the verb, I would suggest moving this text down to: 4.3.4 HTTP PATCH

Reply:
I don't think we should client changes to the TimeMap, I think this should be just a server generated thing (so not PUT or PATCH)

Reply:
Since an LDPCv is an LDPC, if we are disallowing requests that are allowed by LDPCs I would think we would want to specify that.  My reasons for including this here are the same as Accept-Post, so I will move them both if that is preferable.